bundles/org.openhab.automation.pythonscripting/src/main/java/org/openhab/automation/pythonscripting/internal/graal/GraalPythonBindings.java

```diff
@@ -0,0 +1,127 @@
+/*
+ * Copyright (c) 2010-2025 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.automation.pythonscripting.internal.graal;
+
+import java.util.AbstractMap;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+
+import javax.script.ScriptContext;
+import javax.script.ScriptEngine;
+import javax.validation.constraints.NotNull;
```

Comment:
I'm not sure if there is a specific reason for using that here, but generally we use the null annotations from `org.eclipse.jdt.annotation`, e.g. `org.eclipse.jdt.annotation.NonNull`.

Reply:
was introduced by a mistake. But now I checked again and it was no necessary anymore. So it is cleaned now.